go.mod

```diff
@@ -44,7 +44,7 @@ require (
 	github.com/gofrs/uuid v2.1.0+incompatible
 	github.com/golang-migrate/migrate v3.5.4+incompatible
 	github.com/golang/mock v1.3.1
-	github.com/golang/protobuf v1.3.3
+	github.com/golang/protobuf v1.3.5
```

Comment:
None of these files were hand-edited, but this is the key change of the whole PR.